nmstate/libnmstate/api.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We do not need it for this PR, please add it to a different one so I can use it for reference.